service/src/main/java/uk/gov/hmcts/reform/fpl/service/CMODocmosisTemplateDataGenerationService.java

```diff
@@ -138,29 +143,54 @@ private CaseManagementOrder getCaseManagementOrder(CaseData caseData) {
         return null;
```

Comment:
not part of your PR but lines 134-136 contain duplicated code that can be removed

Reply:
Think that was from when @swalker125 merged the 2 different cmo objects that we had

Reply:
done